components/tabs/test/tabs.visual-diff.js

```diff
@@ -7,12 +7,10 @@ describe('d2l-tabs', () => {
 
 	let browser, page;
 
-	const focusTabs = (selector) => {
-		return page.$eval(selector, (elem) => {
-			return new Promise((resolve) => {
-				elem.focus();
-				requestAnimationFrame(resolve);
-			});
+	const focusTabs = async(selector) => {
+		await focusWithKeyboard(page, selector);
+		await page.evaluate(() => {
+			new Promise(resolve => requestAnimationFrame(resolve));
```

Comment:
So, some cases of needing `requestAnimationFrame`...

Reply:
Yeah, not a new case but another case still the same.

Reply:
As you might know, tabs does a lot of stuff when moving focus around, and those things are `async`, which is probably why rAF is needed.